src/borg/testsuite/archiver.py

```diff
@@ -988,6 +988,7 @@ def test_readonly_repository(self):
             # Restore permissions so shutil.rmtree is able to delete it
             os.system('chmod -R u+w ' + self.repository_path)
 
+    @pytest.mark.skipif('BORG_TESTS_UNRELIABLE_MODES' in os.environ, reason='modes unreliable')
```

Comment:
that env var name is a bit confusing.
isn't it rather BORG_TESTS_SKIP_UNRELIABLE_MODES?
"borg tests unreliable modes" sounds rather like the opposite.